HW_1/task_1.py

```diff
@@ -0,0 +1,8 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+n = int(input('Введите трехзначное число: '))
```

Comment:
Чтобы не применять функцию `abs` постоянно, сделайте это один раз.
```suggestion
n = abs(int(input('Введите трехзначное число: ')))
```